SpatialGDK/Documentation/content/technical-overview/how-the-gdk-fits-in.md

```diff
@@ -13,19 +13,16 @@ The diagram below shows how SpatialOS and the GDK fit into a typical multiplayer
 <%(Lightbox image="{{assetRoot}}assets/tech-overview/architecture-of-spatialos-gdk-for-unreal-game.jpg")%><br>
 _The GDK provides a networking integration with SpatialOS, which enables Unreal Engine 4 clients and servers to communicate with the SpatialOS Runtime to synchronize state._
 
-You upload your built-out UE4 server binaries to SpatialOS, which runs them as part of a deployment. You can also upload clients to SpatialOS and distribute them to players using the [SpatialOS Launcher]({{urlRoot}}/content/glossary#launcher) for early playtesting.
-
-You can integrate systems sitting outside the game instance, such as inventory, authentication and matchmaking, using SpatialOS’s [identity and platform services](https://docs.improbable.io/reference/latest/platform-sdk/introduction).
-
-## The GDK in more detail
-In Unreal, game clients communicate with the game server using Unreal’s networking code. The GDK contains a plugin for Unreal Engine that replaces this networking code. You can switch between Unreal networking and SpatialOS networking from the toolbar in the Unreal Editor. 
+## How it works
+In Unreal, game clients communicate with the game server using Unreal’s networking code. The SpatialOS GDK for Unreal contains a plugin for Unreal Engine that replaces this networking code. You can switch between Unreal networking and SpatialOS networking from the toolbar in the Unreal Editor. 
 
 When we forked Unreal Engine, we extended Unreal’s `UIpNetDriver` (which orchestrates replication) to create a `USpatialNetDriver`. This handles the connection between the GDK and SpatialOS, and translates Unreal’s native replication updates and RPCs into instructions that SpatialOS can follow. We do this by using the `UnrealHeaderTool` to generate reflection data that we then turn into the SpatialOS data format called schema.
 
 <%(Lightbox image="{{assetRoot}}assets/tech-overview/networking-switch.png")%><br>
 _Use the Unreal Editor toolbar networking switch to swap out native Unreal networking and swap in SpatialOS networking._
 
-The SpatialOS model differs significantly from Unreal Engine when it comes to replicating an Actor. We don't replicate Actors to each player individually, as Unreal would. Instead, we update the game instance running in the cloud, and it’s SpatialOS which handles distributing this data to connected clients, so data is not sent multiple times to each interested client.
+The SpatialOS model differs significantly from Unreal Engine when it comes to replicating an Actor. We don't replicate Actors to each player individually as Unreal would. Instead, we update the game instance running in the cloud, and SpatialOS distributes the data to connected clients so it is not sent multiple times to each interested client.
```

Comment:
"interested" client is a problematic term. I would avoid "interested" because it's confusion potential is great.

In fact, the SpatialOS Runtime sends updates to clients if they have active read access (which includes having interest in an entity component).

Maybe:
 " ....multiple times to each client." is all we need.

Reply:
Fixed